components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -3957,6 +4030,20 @@ private String generateFilterValueForBackend(String searchOperation, String sear
     @Override
     public int getCountOfApplications(String filter) throws IdentityApplicationManagementException {
 
+        return getCountOfApplications(filter, false);
+    }
+
+    /**
+     * Get count of applications for user which has the filter string
+     *
+     * @param filter
+     * @return
+     * @throws IdentityApplicationManagementException
+     */
```

Comment:
Check the method description for formatting issues.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6026/commits/964c6d3814dba4a93591240fb7b95b4f3b9f77c3